libsplinter/src/admin/service/event/store/error.rs

```diff
@@ -45,3 +48,63 @@ impl fmt::Display for AdminServiceEventStoreError {
         }
     }
 }
+
+impl From<AdminServiceStoreError> for AdminServiceEventStoreError {
```

Comment:
What's the relationship between these errors?

Reply:
The `AdminServiceStore` has `TryFrom` implementations for some of the `CircuitProposal` data. Specifically, the enums that are represented as strings in the database have `TryFrom` implemented, so rather than re-implement here, those TryFroms that are defined in the AdminServiceStore are being used.